dxapp.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "upload_multiqc",
+  "title": "Upload Multiqc",
+  "summary": "Uploads multiqc file to the Viapath Genomics server",
+  "tags": [
+    "QC"
+  ],
+  "dxapi": "1.0.0",
+  "inputSpec": [
+    {
+      "name": "multiqc_html",
+      "label": "Multiqc HTML",
+      "help": "A multiqc html output file",
+      "class": "file",
+      "optional": false
+    }
+  ],
+  "outputSpec": [
+    {
+      "name": "upload_multiqc",
+      "label": "MultiQC server index.html files",
+      "help": "Upload MultiQC app outputs",
+      "class": "array:file"
+    }
+  ],
+  "runSpec": {
+    "file": "src/code.sh",
+    "release": "14.04",
+    "interpreter": "bash",
+    "distribution": "Ubuntu"
+  },
+  "access": {
+    "network": [
+      "*"
+    ]
+  },
+  "details": {
```

Comment:
Now we state the version number in the name of the app this is probably redundant but please change the github tag from details to properties (as below). This is because this custom property is displayed as a column in 001 and may be absent if not added as a property 
  "properties": {
    "github release": "v1.0"
  },

Reply:
This github release tag needs to be within a properties section, not details otherwise not visible in Nexus UI